### DISCUSSION TITLE
Rollback request statechanges when form is dirty/ already filled

### DISCUSSION THREAD

**Discussion by sukrosono** [community/community#7849] [Discussions]
Hello, I just typed for 30 minutes while creating a GitHub discussion. Then recklessly change the type from QA to another.
Poof, shit happen.

my feedback is, just prevent the override on that text field. Current state changes remove my work, Hopefully, this is not happening to future GitHub users. Actually, it happen to twitch tho, the last was I am creating this on other forms, then I realize I should put discussion on the product instead of action. 

Thank you blackcat

**Comment by ralphjsmit** [comment]
Yes, this is something you absolutely don't expect! Please fix!
